.circleci/config.yml

```diff
@@ -49,19 +46,12 @@ jobs:
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
 
       - run: bin/setup
-      - run: bundle exec bin/drake db:test:prepare
```

Comment:
This is from the first commit, just so we have failing tests to check if Circle displays those failing tests correctly.